homework/index.js

```diff
@@ -27,21 +31,136 @@
+    try {
+      const contributors = await fetchJSON(contributorUrl);
+
+      const rightDiv = document.getElementById('cont');
+      rightDiv.innerHTML = '';
+      contributors.forEach(contributor => {
+        const nameOfContributor = contributor.login;
+        const numberOfContributor = contributor.contributions;
+        const avatarUrl = contributor.avatar_url;
+
+        const photoContributor = createAndAppend('div', rightDiv, {
+          class: 'avatar-div',
+        });
+        const infoContributor = createAndAppend('div', rightDiv, { class: 'avatar-div data' });
+
+        createAndAppend('img', photoContributor, { src: avatarUrl });
+        createAndAppend('a', infoContributor, {
+          text: nameOfContributor.toUpperCase(),
+          href: `https://github.com/${nameOfContributor}`,
+          target: '_blank',
+        });
+        createAndAppend('br', infoContributor, {});
+
+        createAndAppend('span', infoContributor, {
+          text: 'Forks: ',
+        });
+        createAndAppend('span', infoContributor, {
+          text: numberOfContributor,
+          class: 'result label',
+        });
+        createAndAppend('br', infoContributor, {});
+      });
+    } catch (error) {
+      const rightDiv = document.getElementById('cont');
+      createAndAppend('div', rightDiv, {
+        text: "there is no contributor's available",
+        class: 'alert-error',
+      });
+    }
+  }
+
+  // reload and render all data after async
+  function reloadAndRenderAllData(repository) {
+    const info = document.getElementById('info');
+    info.innerHTML = '';
+    const repositoryDescription = repository.description;
+    const fork = repository.forks;
+    const updated = repository.updated_at;
+    const dataName = repository.name;
+    const contributorsUrl = repository.contributors_url;
+
+    if (repositoryDescription == null) {
+      const noDescription = createAndAppend('li', info, {});
+      createAndAppend('span', noDescription, {
+        text: 'Description: there is no description',
+        class: '',
+      });
+    } else {
+      const liDescription = createAndAppend('li', info, {});
+      createAndAppend('span', liDescription, { text: 'Description: ' });
+      createAndAppend('span', liDescription, { text: repositoryDescription, class: 'result' });
+    }
+
+    const link = createAndAppend('li', info, {});
+    createAndAppend('span', link, { text: 'link: ' });
+    createAndAppend('a', link, {
+      text: dataName,
+      target: '_blank',
+      href: `https://github.com/HackYourFuture/${dataName}`,
+      class: 'result',
+    });
+    const liOfForks = createAndAppend('li', info, {});
+    createAndAppend('span', liOfForks, { text: 'Forks: ' });
+    createAndAppend('span', liOfForks, { text: fork, class: 'result' });
+    const liUpdateAt = createAndAppend('li', info, {});
+    createAndAppend('span', liUpdateAt, { text: 'updated_at: ' });
```

Comment:
The `updated_at` date which is retrieved from the GitHub API comes in ISO 8601 format i.e 2018-04-06T17:18:06Z. This can be shown in a more readable format like `dd/mm/yyyy`.

For this, I would suggest that you explore the Date object of JavaScript and see if you can make this appear on UI in `dd/mm/yyyy` format - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Date